### PR TITLE
Migrate isolate_state.dart to null safety.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/lib/src/service/isolate_state.dart [hunk 1]
```diff
@@ -13,25 +11,25 @@ import 'package:vm_service/vm_service.dart' hide Error;
...
   IsolateState(this.isolateRef);
 
-  ValueListenable<bool> get isPaused => _isPaused;
+  ValueListenable<bool?> get isPaused => _isPaused;
 
   final IsolateRef isolateRef;
 
-  Future<Isolate> get isolate => _isolate.future;
-  Completer<Isolate> _isolate = Completer();
+  Future<Isolate> get isolate => _completer.future.then((value) => value!);
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/service/isolate_state.dart:18]: _completer may complete with null, so we do not want to add a !. Make the return type `Future<Isolate?>`

**Review by kenzieschmoll — APPROVED** [review]
one comment then lgtm

File: packages/devtools_app/lib/src/service/isolate_state.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/service/isolate_state.dart:18]: done

**Comment by polina-c** [comment]
tested

**Pull request merged**